docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+#Import Python Base Image(change version as needed)
+FROM python:3-slim
+
+#Install the python SDK
+RUN pip install --upgrade ibm-watson==3.2.0
```

Comment:
Keep this to `ibm-watson>=3.2.0`. 

Ideally we would want bumpversion to update it as well, but its ok for now.